source/documentation/rules-g.html.md.erb

```diff
@@ -0,0 +1,1208 @@
+---
+title: NCTS Phase 5 Technical Interface Specification
+weight: 8
+description: Software developers, designers, product owners or business analysts. Integrate your software with the ERMIS service
+---
+#Rules G
+Based on document version 5.15.0-v0.10 and issue date 04/03/2022
+<%= partial 'documentation/partials/rulesintro' %>
+##G0001
+
+<b>Functional Description</b>
+
+If at least one &lt;CONSIGNMENT-HOUSE CONSIGNMENT-ADDITIONAL INFORMATION.Code&gt; is EQUAL to ’30600’ in the declaration, then for the specific &lt;CONSIGNMENT-HOUSE CONSIGNMENT&gt; the Data Group CONSIGNEE shall not be used AND &lt;CONSIGNMENT-CONSIGNEE&gt; shall not be used. For the rest of the repetitions of &lt;CONSIGNMENT-HOUSE CONSIGNMENT&gt; the specific IF statement (“If at least one &lt;CONSIGNMENT-HOUSE CONSIGNMENT-ADDITIONAL INFORMATION.Code&gt; is EQUAL to ’30600’”) shall be re-validated.<br/>During the Transitional Period same approach shall be followed for the &lt;CONSIGNMENT-HOUSE CONSIGNMENT-CONSIGNMENT ITEM-ADDITIONAL INFORMATION.Code&gt;.<br/>
+
+<b>Technical Description</b>
+
+N/A
+
+##G0002
+
+<b>Functional Description</b>
+
+XSD contains a non-standard regular expression for this data item.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0003
+
+<b>Functional Description</b>
+
+This Data Item is omitted if the Data Group &lt;RISK ANALYSIS RESULT&gt; does not refer to a specific Goods Item but is applicable to the whole Consignment.<br/>This Data Item has the value '0' if the Data Group &lt;RISK ANALYSIS RESULT&gt; does not refer to a specific Goods Item but is applicable to all Goods Items included in the Consignment.<br/>This Data Item has a non-zero value if the Data Group &lt;RISK ANALYSIS RESULT&gt; refer to a specific Goods Item.
+
```

Comment:
Grammar issue - `This Data Item has a non-zero value if the Data Group &lt;RISK ANALYSIS RESULT&gt; refer to a specific Goods Item.` needs to use "refers" (not "refer") because group is singular.